lib/tasks/migrations/migrations.rake

```diff
@@ -265,8 +265,8 @@ namespace :migrations do
   def uri(path)
     raise red 'Unable to migrate: No "V3_ENDPOINT" provided, please check your .env file.' unless ENV["V3_ENDPOINT"].present?
 
-    res = URI(ENV["V3_ENDPOINT"])
-    res.path = "/api/v1/migrations/#{path}.json"
+    base_uri = URI(ENV["V3_ENDPOINT"])
+    res = URI::join(base_uri, "/api/v1/migrations/#{path}.json")
```

Comment:
I think this has to `res = URI::join(base_uri, "api/v1/migrations/#{path}.json")`

Reply:
You are right - it will be treated as an absolute path and base_uri will be discarded if it starts with a slash